.github/workflows/autodeps.yml

```diff
@@ -21,6 +21,8 @@ jobs:
     steps:
       - name: Checkout
         uses: actions/checkout@v4
+        with:
+          persist-credentials: true
```

Comment:
I would like comments briefly explaining when `true` vs `false` should be used.

Reply:
```suggestion
        with:
          persist-credentials: true  # credentials are needed to push commits
```